sdk/kusto/arm-kusto/CHANGELOG.md

```diff
@@ -2,13 +2,7 @@
 
 ## 7.0.1 (Unreleased)
 
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+Only minior change and fix the sample generation issues.
```

Comment:
Could you change it into "add generated samples from swagger examples" ?

Reply:
done